### DISCUSSION TITLE
GUI.JS Doesn't work at all. It just disappears.

### DISCUSSION THREAD

**Discussion by Su4o** [3kh0/ext-remover#817] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
Here's a video. 

[Screen recording 2023-11-21 08.40.10.webm](https://github.com/3kh0/ext-remover/assets/127699325/3d085911-28cf-4970-9519-8156976891d0)

Here is my system info:
Google Chrome: Version 118.0.5993.124 (Official Build)  (64-bit)
Platform: 15604.57.0 (Official Build) stable-channel octopus
Channel: stable-channel
Firmware Version: Google_Fleex.11297.296.0
ARC Enabled: true
ARC: 10919957
Enterprise Enrolled: true
Developer Mode: false

**Comment by S-PScripts** [comment]
doesn't seem to be there anymore
